### ISSUE TITLE
[MU4 Issue] Default workspace name not translatable, because logic relies on it

### ISSUE THREAD

**Issue by cbjeukendrup** [musescore/MuseScore#12658]
**Describe the bug**
The name of the default workspace (`"Default"`) is not translatable, and we can't easily make it translatable because there is some logic relying on it (logic to determine which one is the built-in default workspace).

**To Reproduce**
1. See status bar, and see the source code, `workspacetypes.h`

We should make a better way to determine which workspace is the default one, instead of relying on the name. (I don't think we should still have logic relying on the name after we make it translatable. Relying on translatable names is asking for trouble, if you ask me.)

**Comment by cbjeukendrup** [comment]
In fact, the only reason that we need to check whether a workspace is the default one, is that we don't allow deleting the default workspace. Do we really want to disallow this? Or do we rather want the the user can simply not delete the last remaining workspace (i.e. there always has to exist at least one workspace)?

**Comment by Tantacrul** [comment]
@cbjeukendrup - we specified that the default workspace can't be deleted in case users really need it to revert settings they messed up. I don't have opinions on how this should be achieved.

**Comment by Tantacrul** [comment]
yeah, you're right. Oh well then. The option to just enforce '1 workspace at all times' is good.

We'll need to spruce up this system in other releases.
